dashboards-observability/public/components/explorer/home_table/history_table.tsx

```diff
@@ -40,44 +40,100 @@ export function Table(options: TableData) {
   };
 
   const columns = [
+    {
+      field: 'type',
+      name: '',
+      width: '3%',
+      render: (item) => {
+        if (item == 'Visualization') {
+          return (
+            <div>
+              <EuiLoadingChart size="m" />
```

Comment:
Suggestion: This can be `visualizeApp` Icon.

Reply:
It was Ani's choice to have a chart as icon for visualization, that's why went with this